fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -863,16 +864,17 @@ struct RocksDBKeyValueStore : IKeyValueStore {
 		std::shared_ptr<ReadIteratorPool> readIterPool;
 		std::shared_ptr<PerfContextMetrics> perfContextMetrics;
 		int threadIndex;
-		ThreadReturnPromiseStream<std::tuple<int, std::string, double>> metricPromiseStream;
+		ThreadReturnPromiseStream<std::pair<std::string, double>>* metricPromiseStream;
```

Comment:
super nit: can you add a comment what the std::pair represent?

Reply:
Added